using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -110,36 +110,64 @@ The number of seconds to wait before Analytics launch hits are sent from the SDK
 2. Import the Analytics extension in your application's main activity.
 
    ```java
-    import com.adobe.marketing.mobile.*;
+    import com.adobe.marketing.mobile.MobileCore;
+    import com.adobe.marketing.mobile.Analytics;
    ```
 {% endtab %}
 
-{% tab title="iOS" %}
-1. Add the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) and Analytics extensions to your project using Cocoapods.
+{% tab title="iOS (AEP 3.x) %}
```

Comment:
Is this missing a terminating "?:
`{% tab title="iOS (AEP 3.x) %}` -> `{% tab title="iOS (AEP 3.x)" %}`

Reply:
will fix all areas with the same issue.

Reply:
Awesome, thanks Calise!